_episodes/metadata.md

```diff
@@ -3,28 +3,154 @@ title: "Metadata"
+Using an ontology helps:
+
+- others to understand the structure and content of your data,
+- make your data findable, interoperable and reusable.
+
+> ## A controlled vocabulary for climate and forecast data
+>
+> In climate-related domains, many variables depend on the type of surface.
+> How can you specify the surface type in the metadata?
+>
+>> ## Solution
+>>
+>> Climate and Forecast metadata ([CF conventions](http://cfconventions.org/))
+>> maintains a vocabulary specifically for specifying surface and area types.
+>> The vocabulary is available on the CF site as the
+[Area Type Table](http://cfconventions.org/Data/area-type-table/current/build/area-type-table.html).
+> {: .solution}
+{: .challenge}
+
+## Metadata standards (or conventions)
+
+A metadata standard (or convention) is a subject-specific guide to your data.
+Rules on what content must be included, what syntax must be used, or a controlled vocabulary
+are included in a metadata standard.
+The quality of your metadata has a huge impact on the reusability of your research data.
+It is best practices to use metadata standard and/or an ontology commonly used in your community/research team.
+
+Some of the recognized metadata standards for climate-related domains are:
+
+- Climate and Forecast metadata ([CF conventions](http://cfconventions.org/))
+- World Meteorological Organization Core Metadata Profile ([WMO-CMP](https://www.wmo.int/pages/prog/www/WIS/metadata_en.html))
+- Generic Earth Observation Metadata Standard ([GEOMS](https://avdc.gsfc.nasa.gov/index.php?site=1925698559))
+- Cooperative Ocean-Atmosphere Research Data Service Conventions ([COARDS](https://ferret.pmel.noaa.gov/Ferret/documentation/coards-netcdf-conventions))
+- Water Markup Language ([WaterML](https://www.ogc.org/standards/waterml))
+- Shoreline Metadata Profile of the Content Standards for Digital Geospatial Metadata ([SMP-CSDGM](https://www.fgdc.gov/standards/projects/FGDC-standards-projects/metadata/shoreline-metadata/))
+
+> ## A recognized metadata convention for climate data
+>
+> Let's do a search for the keyword `climate` in [FAIR satndards](https://fairsharing.org/standards/).
+>
+> 1. What metadata convention did you find?
+> 2. Which file format does support the implementation of the metadata?
+> 3. What are other domains that use this convention?
+>
+>> ## Solution
+>>
+>> 1. Climate and Forecast metadata ([CF conventions](http://cfconventions.org/))
+>> 2. NetCDF
+>> 3. Atmospheric science, earth science, natural science, and oceanography
+> {: .solution}
+{: .challenge}
+
+> ## no/incomplete metadata standards in your research team
+>
+> Assume that you want to use metadata describing processes that produce data.
+>
+> 1. The available standards in your research team are not specifically suited
+> for that purpose.
+> 2. There is no standard in your research team for that purpose.
+> How can you define a relevant metadata scheme?
```

Comment:
I find this discussion point a bit difficult to follow. It seems like 1 and 2 are very similar.

How about:

```suggestion
> Imagine you are working in a lab, and you want to use metadata describing processes that produce data. However, the available standards in your research team are not specifically suited for that purpose. 
>
> How can you define a relevant metadata scheme?
> What would you do if there was no standard in your research team?
```